docs/devsguide/new_dev_guide.rst

```diff
@@ -209,5 +211,374 @@ text (Vim, TextEdit, Nano, etc.) and doesn’t invisibly add characters to the
+
+For the latest version, in the command line enter ::
+
+	$ pip install sphinxcontrib-bibtex
+
+``Resources:``
+
+#. `Documentation <https://sphinxcontrib-bibtex.readthedocs.io/en/latest/>`_ 
+#. `Known Issues and Workarounds <https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#known-issues-and-workarounds>`_
+#. `Example <https://sphinxcontrib-bibtex.readthedocs.io/en/latest/quickstart.html#minimal-example>`_
+
+
+**PrettyTable:**
+
+PrettyTable is a python library that adds a lot of versatility to table creation.
+
+``To Install:``
+
+For the latest version, in the command line enter ::
+
+	$ pip install prettytable
+
+``Resources:``
+
+#. `Tutorial <https://code.google.com/archive/p/prettytable/wikis/Tutorial.wiki>`_
+
+
+**Cloud Sphinx:**
+
+Cloud is a Sphinx theme that PyNE uses to generate its 
+HTML documentation (like this site).
+
+``To Install:``
+
+For the latest version, in the command line enter ::
+
+	$ pip install cloud_sptheme
+
+``Resources:``
+
+#. `Documentation <https://cloud-sptheme.readthedocs.io/en/latest/>`_
+#. `Source <https://foss.heptapod.net/doc-utils/cloud_sptheme>`_
+
+
+**Jupyter:**
+
+If you have downloaded Python through Anaconda Jupyter requirements should 
+be satasfied, but it never hurts to make sure. 
+
+You can check there version by entering ::
+
+	$jupyter —-version
+
+``To Install:``
+
+If you are going to use Python 2:
+
+For the latest version, in the command line enter ::
+
+	$ pip install jupyter
+
+If you are going to use Python 3:
+
+For the latest version, in the command line enter ::
+
+	$ pip3 install jupyter
+
+``Resources:``
+
+#. `Additional Installation Information <https://jupyter.readthedocs.io/en/latest/install.html>`_
+#. `Project Documentation <https://jupyter.readthedocs.io/en/latest/projects/doc-proj-categories.html#deployment>`_
+
+
+------------
+PyNE Library
+------------
+
+PyNE is what is known as a library in Python (as with many languages, 
+libraries can be built and installed to add functionalities to the language), 
+and it is designed to assist in Nuclear Engineering projects. 
+
+**Python:**
+
+Python is a popular computer language for building applications, 
+data science, and visualization. 
+
+``To Install:``
+
+A comprehensive Python package can be found in Anaconda, which provides a 
+free installation for MacOS, Windows, and Linux. Follow the instructions for your 
+specific platform `here <https://docs.anaconda.com/anaconda/install/>`_ . 
+Installing through Anaconda will result in many of the packages and dependencies 
+for PyNE being satisfied.
+
+If you want to download Python without the rest of Anaconda, use 
+the links in the resources below. Keep in-mind that PyNE is currently 
+not supported on Python 3.7 or later.
+
+``Resources:``
+
+#. `Python Download (Not Anaconda) <https://www.python.org/downloads/>`_
+#. `Python Wiki <https://wiki.python.org/moin/>`_
+#. `Beginner’s Guide <https://wiki.python.org/moin/BeginnersGuide>`_
+
+
+**Ruby:**
```

Comment:
Ruby is only required on MacOS, and only then to install homebrew.  Perhaps there should be platform-specific sections?

Reply:
Thank you for the feedback on my pull request!
I am toying with how to include platorm-specific install instructions for these dependencies. I feel like this document is getting really long, which isn't a bad thing, but it might feel overwhelming for New Developer's. I've moved the instructions for installing ruby into the MacOS instructions for installing git, which also outline how to install homebrew through ruby. 

Do you think it would be appropriate to find links to installation instructions for each of the dependencies, so that each platform is covered? 

Some of the dependencies have this already, but I thought that Homebrew worked for other platforms so I opted to include brew install and pip install (if I'm not mistaken, that's a python tool and doesn't depend on platform) instructions for dependencies that lacked a lot of documentation (or where the documentation I was reading through didn't seem to indicate a difference for the various platforms).

Reply:
I agree that it is challenging to balance being thorough with not being intimidating to new user/developers!

While brew may work on other platforms, it is typically not in the top 3 ways that people install software on those platforms.  One day someone will make the package manager to rule them all, but so far, brew is not that....

I know that tables are difficult to maintain in RST, but I wonder if some kind of table with each row being a different dependency, and each column being a different platform???

Otherwise, we might want to just point folx to different files for each platform (and you might not be responsible for all the platforms.)

Reply:
I see you moved the Ruby section up to the description of installing homebrew.... do you need it here still?

Reply:
I think different files for each platform is a good plan.

Reply:
I've created three separate pages for Mac, Linux, and Windows dependencies which have installation instructions for each platform. These pages are now linked in a section before the one on installing PyNE, and the original section for dependencies was removed.

Reply:
Hm. I'm not sure removing the original list of dependencies is a good idea.

(Also, many mac users don't use homebrew. I, for example, use macports.)